third_party/eigen.BUILD

```diff
@@ -64,6 +64,7 @@ cc_library(
         # This define (mostly) guarantees we don't link any problematic
         # code. We use it, but we do not rely on it, as evidenced above.
         "EIGEN_MPL2_ONLY",
+        "EIGEN_MAX_ALIGN_BYTES=64",
```

Comment:
This may have significant performance impact. We cannot just change this globally.

Reply:
This change was made following the suggestions in this [comment](https://github.com/tensorflow/tensorflow/pull/15606#issuecomment-361359663) in the previous PR, and also because it fixes the crashes.

Newbie question here, but what is the alternative to setting this globally?  Perhaps try and figure out why EIGEN_MAX_ALIGN_BYTES=0 does not seem do the correct thing?

Reply:
@markdryan we are investigating some of the unit test failures with Eigen/AVX512 as well. https://github.com/tensorflow/tensorflow/issues/18676 is an issue with Eigen FP16 matmul.